src/components/Section/Identification/ApplicantBirthDate/ApplicantBirthDate.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { i18n } from 'config'
-import { TODAY } from 'constants/dateLimit'
+import { TODAY } from '../../../../constants/dateLimit'
```

Comment:
i think it should be `constants/dateLimits`. You also need to `export` the `const` from `constants/dateLimits`.

Reply:
ya, I just fixed it, sorry
I forgot that I had exported it in my other branch and not this one